config/routes.rb

```diff
@@ -9,9 +9,10 @@
 
   namespace :v0, defaults: { format: 'json' } do
     resources :in_progress_forms, only: [:index, :show, :update, :destroy]
-    resources :letters, only: [:index, :show] do
+    resources :letters, only: [:index] do
       collection do
         get 'beneficiary', to: 'letters#beneficiary'
+        post ':id', to: 'letters#download'
```

Comment:
The Benefit Summary Letter (BSL) includes a [JSON body of options](https://github.com/department-of-veterans-affairs/vets-api/blob/1cdbb16fb1424154e4717b447ecdf38fce610ab6/spec/lib/evss/letters/service_spec.rb#L76-L88). Rather than have two endpoints all letters use POST (with a blank body for the others). Note that the options are not strictly for the BSL other letters could use them in the future.